app/src/org/commcare/network/RemoteDataPullResponse.java

```
@@ -128,4 +129,8 @@ protected long guessDataSize() {
         }
         return -1;
     }
+
+    public String getRetryHeaderValue() {
+        return response.getFirstHeader("Retry-After").getValue();
```

Assuming we are fine with this failing hard if the header value isn't around